tests/stress_test/README.md

```diff
@@ -0,0 +1,32 @@
+# EOS STRESS TEST SCRIPT
+
+
+The following stress test script will send bunch of transfer transactions equally distributed to each node in the testnet:
+ - transfer from inita to initb
+ - transfer from initb to initc
+ - transfer from initc to initd
+ - .....
+ - transfer from initu to inita
+
```

Comment:
Only inita and initb is in the genesis file. initc, initd, .. initu are not in the genesis file.

Reply:
We have decided to go ahead and add initc..initu so this test will work.